resources/dicts/conditions/pregnancy.json

```
@@ -90,7 +109,11 @@
       "m_c secretly had a {insert} with r_c.",
       "m_c hopes that {PRONOUN/m_c/poss} {insert} doesn't look too much like r_c, otherwise questions might follow.",
       "r_c goes to visit m_c in the nursery with {PRONOUN/m_c/poss} new {insert}, on a completely innocent mission to deliver food to the new parent.",
-      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. "
+      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. ",
+	    "No one wanted to ruin such a happy occasion but... it didn't take a genius to notice how m_c's {insert} looked suspiciously like r_c when they shouldn't...",
```

```suggestion
	    "No one wanted to ruin such a happy occasion but... it didn't take a genius to notice how m_c's {insert} looked suspiciously like r_c...",
```

---

```suggestion
	    "No one wants to ruin such a happy occasion, but... it doesn't take a genius to notice how m_c's {insert} looks suspiciously like r_c...",
```
i think these are supposed to be in present tense?